### ISSUE TITLE
order dir list by last modified desc

### ISSUE THREAD

**Issue by mhewedy** [sevdokimov/log-viewer#158]
current implementation order ascending by natural order...
It is fine ... but the user might be interested in files that are more recent to be on top (last modified desc)

What do you think?

I can send PR once agreed upon!

**Comment by sevdokimov** [comment]
Agree, order my modification time is good in some cases. We need to add clickable column names or a dropdown with "name", "modification time", "size" options to specify sorting.

**Comment by holograph** [comment]
Clickable columns would be ideal, but since this _is_ a log viewer, I think a default sort order of modified time (descending) makes sense in the interim...

P.S. I just integrated logviewer for a client, it's far better than any alternative I've found so far. Thank you!
